sync.cmd

```diff
@@ -0,0 +1,25 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+set _args=%*
+if /I [%1]==[/?] goto Usage
+if [%1]==[]  set _args=-p
+
+echo Running executor.cmd %_args%
+call executor.cmd %_args%
```

Comment:
How does it know it is supposed to run sync? I would have expected something like `executor.cmd sync %*`